README.md

```diff
@@ -63,7 +63,7 @@ First of all, clone the Git repository:
 Then run:
 
     $ mkdir _build && cd _build
-    $ meson ..
+    $ meson ..    # sometimes it's "meson.py"
```

Comment:
An MSYS2 runtime cannot be detected with `os.name, sys.platform` for the native "/mingw{32,64}" prefixes of MSYS2 and their Pythons. I think that's by design. Native stuff is supposed to be theoretically able to run without the stuff in the Cygwin "/" prefix, so Python there looks like Windows Python.

So MSYS2 ought to patch away the superfluous .py, or write a compatibility script. It's not really a problem that meson should fix, IMO.

I'll make it "on Windows".